doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,6 +185,19 @@ if transceiver is not present:
  - All the workflows mentioned above will reamin same ( or get exercised) till host_tx_ready field update
  - xcvrd will not perform any action on receiving host_tx_ready field update 
 
+# SFF compliant transceiver initialization
+1. Add new flag 'skip_non_cmis_mgr' and platform/hwsku to set this appropriately indicating whether it's supporting determinstic link bring-up sequence (per this document) or not. Default to be set to true for SFF (i.e. non-CMIS) compliant transeivers
+2. xcvrd to do the following as part of its main thread (for initialization/boot-up/restart scenarios)
+   - Spawn a new worker thread SFFManagerTask() to handle SFF (aka non-CMIS) transceivers (optical modules)
```

Comment:
@shyam77git can we have a separate HLD for SFF manager where you can list the objectives:-

1. Deterministic link bring up for SFF compliant transceivers
2. Custom SI settings
3. Enabling High power mode for optics like 100G-DR4